src/components/AdminPage/CardGrid.js

```diff
@@ -49,7 +49,7 @@ export class CardGrid extends Component {
 
   render() {
     const sortOptions = [
-      {key: 'Alphabetically', sort: this.sortByAlphabet, disabled: false}
+      {key: 'A-Z', sort: this.sortByAlphabet, disabled: false}
```

Comment:
Why are you making this change here?

Reply:
Shoot, got my git branches tangled up a bit. This was supposed to be it's own PR, addressing #320 , in order to make that label consistent between pages. I will take this out and make it it's own PR